interface/language/lang_definition.php

```diff
@@ -42,20 +42,24 @@
 
 ?>
 
-<form name='filterform' id='filterform' method='post' action='?m=definition&csrf_token_form=<?php echo attr_url(CsrfUtils::collectCsrfToken()); ?>' onsubmit="return top.restoreSession()">
+<form name='filterform' id='filterform' method='post'
+      action='?m=definition&csrf_token_form=<?php echo attr_url(CsrfUtils::collectCsrfToken()); ?>'
+      onsubmit="return top.restoreSession()">
```

Comment:
In future, rec avoiding changes like this. For example, it appears @tywrenn 's editor will do the opposite of this and revert this change if he edits this script. Thus starting a infinite war of code changes :)

Reply:
ok, it's actually a `PSR12` warning, like the below which is going to be a bear if it progresses to error stage
```
  56 | WARNING | Line exceeds 120 characters; contains 136 characters
```